### PR TITLE
chore(gosec): fix gosec warning

### PR DESCRIPTION
**Pull request by georgettica**
related to [OSD-10161](https://issues.redhat.com/browse/OSD-10161)

### CODE REVIEW

File: pkg/aws/aws.go [hunk 1]
```diff
@@ -76,15 +77,18 @@ func NewClient(accessID, accessSecret, token, region string) (AwsClient, error)
 
 // NewClientFrmFileCredentials creates a new client by reading credentials from a file
 func NewClientFrmFileCredentials(dir string, region string) (AwsClient, error) {
-	filepath := strings.TrimSuffix(dir, "/")
+	dir = strings.TrimSuffix(dir, "/")
+	dir = filepath.Clean(dir)
```
Comment (ramonbutter) [inline pkg/aws/aws.go:81]: can we not leave this one out, and only do it once in 83?

Comment (ramonbutter) [reply pkg/aws/aws.go:81]: and line 80 may also be unrequired, as the 'path.Join' should take care of it how to concatenate the path appropriately. But I will test it

**Comment by georgettica** [comment]
@ramonbutter this is a low prio ticket that wasn't pulled into the sprint
/hold
unless you want to do it regardless

**Comment by ramonbutter** [comment]
It's ok I guess, you can merge it. My comments were just addressing minor details (that were part of my earlier commits) and I can have a look at it later on

**Comment by georgettica** [comment]
Then I'll merge
/unhold

**Comment by georgettica** [comment]
related to OSD-10161

**Pull request merged**